crypto/kzg/src/lib.rs

```diff
@@ -13,13 +13,21 @@ pub use c_kzg::{
     Blob, Bytes32, Bytes48, KzgSettings, BYTES_PER_BLOB, BYTES_PER_COMMITMENT,
     BYTES_PER_FIELD_ELEMENT, BYTES_PER_PROOF, FIELD_ELEMENTS_PER_BLOB,
 };
-pub use c_kzg::{Cell, CELLS_PER_EXT_BLOB};
```

Comment:
Note: the only type which I've changed/removed is the Cell type -- The Blob types etc that are needed for 4844 are left unchanged along with their codepaths. 

We convert from c-kzg Blob type to PeerDASKZG blob type inside of this file.

This is just so that we can be sure that the old c-kzg code has been left untouched.